api/serverpb/server.proto

```diff
@@ -28,12 +28,68 @@ import "protoc-gen-swagger/options/annotations.proto";
 message VersionRequest {}
 
 message VersionResponse {
-  // Full PMM version.
+  // Managed describes pmm-managed build information.
+  message Managed {
```

Comment:
Is it possible to have different version of pmm-managed and pmm-server?

Reply:
Yes, during update. This field describes version embedded into pmm-managed binary during the build. Other fields describe version from RPM.
But this field exists mostly to debug build problems – and we actually found several thanks to it.